CMakeLists.txt

```diff
@@ -952,6 +952,10 @@ if (BUILD_TESTS)
       TEST unit_test SOURCES UnitTest.cpp
       TEST uri_test SOURCES UriTest.cpp
       TEST varint_test SOURCES VarintTest.cpp
+
+    DIRECTORY tracing/test/
+      #Only meaningful on Linux with its bpf() system call
+      #TEST static_tracepoint_test SOURCES StaticTracepointTest.cpp StaticTracepointTestModule.cpp
```

Comment:
Possible now to enable this but to mark this `WINDOWS_DISABLED APPLE_DISABLED`?

Reply:
Just rebased on master and found APPLED_DISABLED.   Oddly, 

WINDOWS_DISABLED APPLE_DISABLED

results in an error saying that WINDOWS_DISABLED is not defined, but 

APPLE_DISABLED WINDOWS_DISABLED 

works.    Also, the comment appeared to cause a problem.